store/file/codec_test.go

```diff
@@ -0,0 +1,83 @@
+package file
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/klauspost/reedsolomon"
+	"github.com/stretchr/testify/require"
+
+	"github.com/celestiaorg/celestia-node/share/sharetest"
+)
+
+func BenchmarkCodec(b *testing.B) {
+	minSize, maxSize := 32, 128
+
+	for size := minSize; size <= maxSize; size *= 2 {
```

Comment:
IMO feels like this should be inside every (sub-) benchmark to make them isolated and more visible relation to `size` variable.

Reply:
Ah...we use it in benchmark name...so (probably) ignore my comment.